docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,37 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
+Documentation: https://chanzuckerberg.github.io/cellxgene-census/
+Contact: cellxgene@chanzuckerberg.com
+ManagedBy: "[Chan Zuckerberg Initiative](http://www.chanzuckerberg.com/)"
+UpdateFrequency: New releases are published weekly. LTS releases are published every 6 months.
+Tags:
+  - single-cell
+  - genomics
+  - computational biology
+License: MIT License
+Resources:
+  - Description: CELLxGENE Census Data
+    ARN: arn:aws:s3:::cellxgene-census/cell-census
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
```

Comment:
I wonder if these are present in any particular order?  I suggest leading with introductory tutorials, and put the complex stuff (e.g., the "online algo" notebook) later

Reply:
These are in the same order as the docsite, but I don't think you're wrong. I'll try to rearrange them.

Reply:
Just realized that I pushed an old version. The newer version has all the notebooks from the docsite. I think re-arranging them is still a good idea, and we may want to do the same on the tutorials page.